test/jdk/com/sun/management/ThreadMXBean/ThreadAllocatedMemory.java

```diff
@@ -298,22 +367,22 @@ public void run() {
             ThreadAllocatedMemory.doit();
             long size2 = mbean.getThreadAllocatedBytes(getId());
 
-            System.out.println(getName() + ": " +
-                "ThreadAllocatedBytes  = " + size1 +
-                " ThreadAllocatedBytes  = " + size2);
+            System.out.println(getName() +
+                " ThreadAllocatedBytes before = " + size1 +
+                " ThreadAllocatedBytes after = " + size2);
 
             if (size1 > size2) {
                 throw new RuntimeException(getName() +
-                    " ThreadAllocatedBytes = " + size1 +
-                    " > ThreadAllocatedBytes = " + size2);
+                    " ThreadAllocatedBytes before = " + size1 +
+                    " > ThreadAllocatedBytes after = " + size2);
```

Comment:
Is this replaceable with `checkResult(...)`?

Reply:
Yes. Done.